invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/LinksSearchItem.js

```diff
@@ -14,12 +14,31 @@ import { AccessDropdown } from "./AccessDropdown";
 import _truncate from "lodash/truncate";
 import { isEmpty } from "lodash";
 import { withCancel, http } from "react-invenio-forms";
+import { dropdownOptions } from "./LinksSearchResultContainer";
 
 export const LinksSearchItem = ({ result, record, fetchData }) => {
   const [copied, setCopied] = useState(false);
   const [loading, setLoading] = useState(false);
   var cancellableAction = undefined;
 
+  const dropdownOptionsGenerator = (value) => {
```

Comment:
please check the comment above

Reply:
there is still code duplication with the file above, the generator of options should be in one place

Reply:
Updated, see here: https://github.com/inveniosoftware/invenio-app-rdm/pull/2607